cumulus/parachains/runtimes/collectives/collectives-westend/src/secretary/mod.rs

```diff
@@ -0,0 +1,232 @@
+	type Preimages = Preimage;
+}
+
+pub type SecretaryCollectiveInstance = pallet_ranked_collective::Instance3;
+
+impl pallet_ranked_collective::Config<SecretaryCollectiveInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_ranked_collective_secretary_collective::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = MapSuccess<Self::PromoteOrigin, ReplaceWithDefault<()>>;
+	type RemoveOrigin = ApproveOrigin;
+	type PromoteOrigin = ApproveOrigin;
+	type DemoteOrigin = ApproveOrigin;
+	type ExchangeOrigin = OpenGovOrFellow;
+	type Polls = SecretaryReferenda;
+	type MinRankOfClass = Identity;
+	type MemberSwappedHandler = (crate::SecretaryCore, crate::SecretarySalary);
+	type VoteWeight = pallet_ranked_collective::Geometric;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkSetup = (crate::SecretaryCore, crate::SecretarySalary);
+}
+
+pub type SecretaryCoreInstance = pallet_core_fellowship::Instance3;
+
+impl pallet_core_fellowship::Config<SecretaryCoreInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_core_fellowship_secretary_core::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Members = pallet_ranked_collective::Pallet<Runtime, SecretaryCollectiveInstance>;
+	type Balance = Balance;
+	type ParamsOrigin = OpenGovOrFellow;
+	// Induction is by any of:
+	// - Root;
+	// - FellowshipAdmin (i.e. token holder referendum);
+	// - A single Member of the Technical Fellowship, rank 3 and above;
+	// - A single member of the Secretary Collective.
+	type InductOrigin = EitherOfDiverse<
+		EitherOfDiverse<
+			EnsureRoot<AccountId>,
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+		>,
+		EitherOfDiverse<
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				FellowshipCollectiveInstance,
+				{ DAN_3 },
+			>,
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				SecretaryCollectiveInstance,
+				{ ranks::SECRETARY },
+			>,
+		>,
+	>;
+	type ApproveOrigin = ApproveOrigin;
+	type PromoteOrigin = ApproveOrigin;
+	type EvidenceSize = ConstU32<65536>;
+	type MaxRank = ConstU32<1>;
+}
+
+pub type SecretarySalaryInstance = pallet_salary::Instance3;
+
+parameter_types! {
+	// The interior location on AssetHub for the paying account. This is the Secretary Salary
+	// pallet instance. This sovereign account will need funding.
+	pub SecretarySalaryInteriorLocation: InteriorLocation = PalletInstance(94).into();
+}
+
+/// [`PayOverXcm`] setup to pay the Secretary salary on the AssetHub in USDT.
+pub type SecretarySalaryPaymaster = PayOverXcm<
+	SecretarySalaryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	AccountId,
+	(),
+	ConvertToValue<WndAssetHub>,
+	AliasesIntoAccountId32<(), AccountId>,
+>;
+
+pub struct SalaryForRank;
+impl GetSalary<u16, AccountId, Balance> for SalaryForRank {
+	fn get_salary(rank: u16, _who: &AccountId) -> Balance {
+		if rank == 1 {
+			1000
```

Comment:
```suggestion
			100 * CENTS
```
Not sure about the exact values, just some mocked ones for testing. 